apollo-router/src/plugins/telemetry/reload.rs

```diff
@@ -22,14 +32,24 @@ use crate::plugins::telemetry::metrics;
 use crate::plugins::telemetry::metrics::layer::MetricsLayer;
 use crate::plugins::telemetry::tracing::reload::ReloadTracer;
 
-type LayeredTracer = Layered<OpenTelemetryLayer<Registry, ReloadTracer<Tracer>>, Registry>;
+pub(super) type LayeredTracer = Layered<
+    Filtered<OpenTelemetryLayer<Registry, ReloadTracer<Tracer>>, SamplingFilter, Registry>,
+    Registry,
+>;
 
 // These handles allow hot tracing of layers. They have complex type definitions because tracing has
 // generic types in the layer definition.
 pub(super) static OPENTELEMETRY_TRACER_HANDLE: OnceCell<
     ReloadTracer<opentelemetry::sdk::trace::Tracer>,
 > = OnceCell::new();
 
+static FMT_LAYER_HANDLE: OnceCell<
+    Handle<Box<dyn Layer<LayeredTracer> + Send + Sync>, LayeredTracer>,
+> = OnceCell::new();
+
+pub(super) static SPAN_SAMPLING_RATE: AtomicU64 =
+    AtomicU64::new(unsafe { mem::transmute::<f64, u64>(0.0) });
```

Comment:
```suggestion
    AtomicU64::new(0);
```
You'll also be able to delete line 1.